source/_integrations/zha.markdown

```diff
@@ -426,11 +426,13 @@ Tip to new users is that, while there is no official list of supported devices,
 
-Such a custom "ZHA Device Handler" are Python scripts that internally are also referred to as a "quirk" because they fix "quirks", like deviations from the standard specifications. ZHA Device Handles do this by transparently, acting as a translator, translating and converting non-compliant device messages and instead present them to the application as coming from a virtual compliant device. These ZHA Device Handlers for Home Assistant can thus be used to parse custom messages to and from Zigbee devices. The ZHA Device Handlers that are made can then be reused by all users in future versions of Home Assistant.
+The few devices that will for example not join/pair properly or while joined/paired do not present all attributes in the ZHA integration likely deviate from the Zigbee specifications set by the [CSA (Connectivity Standards Alliance, formerly the Zigbee Alliance)](https://csa-iot.org/all-solutions/zigbee/) and may therefore require proper bug reports with debug logs from the end-users and as a device owner you may need to actively assistant troubleshooting by providing the information developers need to code custom [ZHA Device Handlers)](https://github.com/zigpy/zha-device-handlers/blob/dev/README.md) that convert/translate non-compliant device functions so they will work properly with the ZHA integration.
 
-The custom quirks implementations for zigpy implemented as ZHA Device Handlers for Home Assistant are a similar concept to that of [Hub-connected Device Handlers for the SmartThings Classics platform](https://docs.smartthings.com/en/latest/device-type-developers-guide/) as well as that of [Zigbee-Herdsman Converters (formerly Zigbee-Shepherd Converters) as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html), meaning they are each virtual representations of a physical device that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms.
+Such a custom "ZHA Device Handler" are Python script that internally is also referred to as a "quirk" because they fix or workaround "quirks" in devices, like deviations from the standard specifications. ZHA Device Handles do this by transparently, acting as an in-line translator/converter, translating and converting non-compliant device messages and instead presenting them to the application as coming from a virtual compliant device that follows the official Zigbee standard. These ZHA Device Handlers for Home Assistant can thus be used to parse custom messages to and from Zigbee devices. Once a quirk is coded, the ZHA Device Handlers that are made can then be reused by all users with the same devices in future versions of Home Assistant.
```

Comment:
This whole chapter is repeating all the above, in different wording, and can thus be removed without losing information.

```suggestion
```